### ISSUE TITLE
Build optimizer and service worker

### ISSUE THREAD

**Issue by xban1x** [angular/angular-cli#7328]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
@angular/cli: 1.3.0-rc.5
node: 8.2.1
os: darwin x64
@angular/animations: 4.2.5
@angular/cdk: 2.0.0-beta.8
@angular/common: 4.2.5
@angular/compiler: 4.2.5
@angular/core: 4.2.5
@angular/flex-layout: 2.0.0-beta.8
@angular/forms: 4.2.5
@angular/http: 4.2.5
@angular/material: 2.0.0-beta.8
@angular/platform-browser: 4.2.5
@angular/platform-browser-dynamic: 4.2.5
@angular/router: 4.2.5
@angular/service-worker: 1.0.0-beta.16
@angular/cli: 1.3.0-rc.5
@angular/compiler-cli: 4.2.5

### Repro steps.
Use `ng build --prod --build-optimizer` flag with serviceWorker set to true

### The log given by the failure.
No log.

### Desired functionality.
For serviceWorker to register normally with build optimizer.

**Comment by RicardoVaranda** [comment]
Hi @xban1x could you please update your @angular/cli to the latest 1.3.0, this issue was fixed in the latest release.

**Comment by xban1x** [comment]
Fixed in 1.3.0
